api/SVGLineElement.json

```diff
@@ -0,0 +1,240 @@
+{
+  "api": {
+    "SVGImageElement": {
+      "__compat": {
+        "mdn_url": "https://developer.mozilla.org/docs/Web/API/SVGImageElement",
```

Comment:
I think this should be `https://developer.mozilla.org/docs/Web/API/SVGLineElement`